trade-finance-manager-ui/templates/case/amendments/amendment-bank-review-date.njk

```diff
@@ -0,0 +1,104 @@
+{% from "govuk/components/date-input/macro.njk" import govukDateInput %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+
+{% import './_macros/amendment-what-is-a-bank-review-date.njk' as whatIsABankReviewDate %}
+
+{% extends "index.njk" %}
+
+{% block pageTitle -%}
+  Amendment - Enter the bank review date
+{%- endblock %}
+
+{% block sub_content %}
+
+{% if error %}
+  {{ govukErrorSummary({
+     titleText: "There is a problem",
+     errorList: error.summary,
+     attributes: {
+       'data-cy': 'amendment--error-summary'
+     },
+     classes: "govuk-!-margin-top-4 govuk-!-margin-bottom-4"
+  }) }}
+ {% endif %}
+
+<form method="POST" autocomplete="off">
+  <input type="hidden" name="_csrf" value="{{ csrfToken }}">
+
+  <fieldset class="govuk-fieldset" role="group" aria-describedby="bankReviewDate-hint">
```

Comment:
Assume the `-hint` is automatically added in the html?

Reply:
Checked together offline, all good and it is added